posts/2021-01-21-jwtbuilder-redirects-21001.adoc

```diff
@@ -52,29 +52,14 @@ In link:{url-about}[Open Liberty] 21.0.0.1:
 
 * <<jwtbuilder, Scheduled payloads with JWT Builder>>
 * <<redirects, Relative linking for HTTP Redirects>>
+* <<conf, MicroProfile Config 2.0 enters general availability>>
 * <<bugs, Significant bugs fixed in this release>>
```

Comment:
Should be "Notable" instead of "Significant" (and we should update this in the template as well).

Reply:
Updated within draft, thanks for spotting.